File: beecrowd/go/bee1165.go
package main

import "fmt"

func main() {
	var n int

	fmt.Scanf("%d", &n)
	for i := 0; i < n; i++ {
		var result bool = true
		var number int = 0

		fmt.Scanf("%d", &number)

		for i := 2; i*i <= number; i++ {
			if number%i == 0 {
				result = false
			}
		}

		fmt.Print(number)
		if result {
			fmt.Println(" eh primo")
		} else {
			fmt.Println(" nao eh primo")
		}
	}

}

File: beecrowd/c/bee1165.c
#include <math.h>
#include <stdio.h>

int is_prime(int num) {
    if (num < 2) {
        return 0;
    }
    if (num == 2) {
        return 1;
    }
    if (num % 2 == 0) {
        return 0;
    }
    for (int i = 3; i <= sqrt(num); i += 2) {
        if (num % i == 0) {
            return 0;
        }
    }

    return 1;
}

int main() {
    int N = 0, inp = 0;
    scanf("%d", &N);

    for (int i = 0; i < N; i++) {
        scanf("%d", &inp);
        printf("%d %s\n", inp, (is_prime(inp) ? "eh primo" : "nao eh primo"));
    }

    return 0;
}
